### ISSUE TITLE
Posts with too many comments choke

### ISSUE THREAD

**Issue by mikekasprzak** [ludumdare/ludumdare#1887]
Example:

https://ldjam.com/events/ludum-dare/46/tiny-fragments

This game has lots of comments. There's a massive batch query that happens: for all comments, fetch each author. In this case, that batch query broke the 250 node fetch limit.

I've since bumped the limit to 500, but this page is incredibly slow loading. The comment renderer is going to need to be reworked.

**Comment by MHillier98** [comment]
Funny timing that I just checked for an issue about comments - I've been having a lot of issues with them not loading and generally being stuck on the spinner, even on my own game (https://ldjam.com/events/ludum-dare/46/feed-the-pillar).

I find that often logging out, clearing cookies, and re-logging in will help. Not sure if that's always required, but it does sometimes work for me.

**Comment by mikekasprzak** [comment]
I believe I pushed a fix for the issue you were running in to, but let me know otherwise.

**Comment by MHillier98** [comment]
Ah thanks, I was just looking through the commits - it happened to me right before I posted my comment, but the change may have just been propagating or something? Will let you know if it happens again.

**Comment by mikekasprzak** [comment]
Yeah it would require hard refresh to be 100% sure you're running the latest code (i.e. in Chrome you push F12, right click the reload button, then choose "Empty Cache and Hard Reload).
